less/about/preferences.less

```diff
@@ -455,6 +455,46 @@ table.sortableTable {
   overflow-y: scroll;
 }
 
+.modal .dialog.paymentHistory .sectionTitle {
+  text-align: left;
+}
+
+span.paymentHistoryButton.browserButton {
```

Comment:
Nice catch!
BTW, this screenshot points out another issue: The currency for the balance is BTC when the wallet is empty, and always USD when non-empty (just confirmed locally).